### DISCUSSION TITLE
Swap ctrl ⌃ and cmd ⌘ keys on MacOS

### DISCUSSION THREAD

**Discussion by hasantahir** [ahrm/sioyek#1203] [Q&A]
Hi,

From my understanding, the new `D-` option maps to the cmd key on MacOS. This understanding is based on https://github.com/ahrm/sioyek/issues/972#issue-2130311645 and https://github.com/ahrm/sioyek/issues/1067#issue-2255808411. To test this, I added a lineL
```
# Close the current sioyek window
close_window <D-w>
``` 
in my `keys_user.config` file but unfortunately, I still need to press `ctrl + w` to close a window. Can you please advise what could be wrong? I am on the latest development branch obtained from [here](https://github.com/hexomancer/sioyek/releases/tag/manual_release19).

Thanks for any help.

**Comment by ahrm** [comment]
I don't have access to a mac right now, maybe someone else could help.

**Comment by nickel8448** [comment]
I don't have a solution.. but I have been experiencing the same thing.
I have kind of become accustomed to use Ctrl + w.
